app/workers/publish_links_worker.rb

```diff
@@ -0,0 +1,15 @@
+class PublishLinksWorker
+  include Sidekiq::Worker
+
+  sidekiq_options retry: 5
```

Comment:
What do you think of implementing the hook `sidekiq_retries_exhausted` and notity Airbrake with the job details? The hook gets called when we exhausted all retries, which could be helpful for debugging purposes.

I'm mentioning it because it saved me a few times in the past!